canvas_modules/common-canvas/src/common-properties/controls/toggle/toggle.scss

```diff
@@ -14,10 +14,16 @@
  * limitations under the License.
  */
 
+.properties-toggle {
+	.properties-label-container {
+		padding-bottom: 0;
+	}
+}
 .properties-table-cell-control { // overrides when in a table cell
-	.properties-toggle {
-		.bx--toggle__switch { // override carbon when inside a table
-			margin: $spacing-02 0;
-		}
+	.bx--toggle__switch { // override carbon when inside a table
+		margin-bottom: $spacing-05;
+	}
+	.properties-toggle .properties-control-item {
+		display: none;
```

Comment:
Instead of using css to hide this label, this label should not have been created at all if inside a table